pdr_backend/cli/cli_arguments.py

```diff
@@ -112,6 +137,54 @@ def add_argument_LOOKBACK(self):
         )
 
 
+@enforce_types
+def check_positive(value):
+    try:
+        ivalue = int(value)
+        if ivalue <= 0:
+            raise Exception("Zero or below.")
```

Comment:
I think it makes sense to separate the conditions for a better readability.

I.e. try the int conversion, throw a type error if it fails specifying that it's not a valid int.
then do the comparison and throw a type error if it is negative

=> two separate, independent and easy to read try/catch blocks